@here/harp-mapview/lib/Utils.ts

```diff
@@ -410,6 +418,242 @@ export namespace MapViewUtils {
         return result;
     }
 
+    /**
+     * Add offset to geo points for minimal vieing box.
+     *
+     * In flat projection, with wrap around enabled, we should detect clusters of points around that
+     * wrap antimeridian.
+     *
+     * Here, we fit points into minimal geo box taking world wrapping into account.
+     */
+    export function wrapGeoPointsToScreen(points: GeoCoordLike[]): GeoCoordinates[] {
```

Comment:
Do we need to expose this?
MapViewUtils is one of the worst parts of our public API b/c it constantly breaks.

Reply:
It's exported for unit-tests only. I can document it as @hidden but still i think it's worth testing.